doc/hardware/porting/custom.rst

```diff
@@ -0,0 +1,16 @@
+.. _platform_cusomizations:
+
+Hardware with Custom Interfaces
+###############################
+
+When the provided support in Zephyr does not cover certain hardware platforms,
+some interfaces can be implemented directly by the platform or a custom
+architecture interface can be used and resulting in more flexibility and
```

Comment:
".. can be used and resulting .."
This whole sentence is a bit too tough to read.